app/_src/quickstart/universal-demo.md

```diff
@@ -100,20 +101,25 @@ kuma-dp run \
           kuma.io/service: redis
           kuma.io/protocol: tcp
     admin:
-      port: 9901"
+      port: 9902"
 ```
 
-For the demo app:
+You can notice that we manually specify readiness port with env variable `KUMA_READINESS_PORT` when every data plane is 
+running on separate machines this is obsolete. 
 
-```sh
-kuma-dp run \
+[//]: # (TODO should we tell users explicitely to open another terminal window or should we just run kuma-dp in background?)
```

Comment:
yeah I'm not sure. I personally like the additional terminal option. We can have the user print maybe the PWD and reference it so they can copy paste it?

Reply:
I also like addition terminals option, but with this you need constantly update path to Kuma binaries directory. I think it would be best if you could simply copy and paste instruction into the terminal and expect everything to work, like for k8s

Reply:
Unfortunately I don't think it's possible without a "navigate to the directory you downloaded the binaries in" instruction

Reply:
Probably we add a comment above this `kuma-dp run` as follows?
```sh
# make sure you've added the Kuma binary directory into your PATH
# make sure your working directory is at the same place at which you place the dataplane manifest
```